pennylane_lightning/lightning_tensor/_measurements.py

```diff
@@ -83,6 +83,28 @@ def expval(self, measurementprocess: MeasurementProcess):
         )._ob(measurementprocess.obs)
         return self._measurement_lightning.expval(ob_serialized)
 
+    def var(self, measurementprocess: MeasurementProcess):
+        """Variance of the supplied observable contained in the MeasurementProcess.
+
+        Args:
+            measurementprocess (StateMeasurement): measurement to apply to the state
+
+        Returns:
+            Variance of the observable
+        """
+
+        if isinstance(measurementprocess.obs, qml.SparseHamiltonian):
+            raise NotImplementedError("Sparse Hamiltonian Observables are not supported.")
```

Comment:
Could you add a test for this unsupported feature?

Reply:
I think this is a false positive complain